libclamav/matcher-pcre.c

```diff
@@ -735,7 +725,7 @@ cl_error_t cli_pcre_scanbuf(const unsigned char *buffer, uint32_t length, const
                             ret = CL_EMEM;
                             break;
                         }
-                        newres->virname    = pm->virname;
+                        newres->virname    = "test";
```

Comment:
Yup!  It's used for testing signatures with sigtool or with the matcher unit tests for signature support.  I had to pick some static name because the actual signature name isn't available in this situation. 
In the future I'd like to change how sigtool works for testing signatures -- that is to test whole LDB signatures instead of testing each subsignature a la carte.  Then we could remove this condition entirely.  But I'm not sure how much work that would be and didn't have time to do it now.